flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java

```diff
@@ -131,6 +131,7 @@ private static void syncDatabase(
         String multiToOneTarget = params.get("multi-to-one-target");
         boolean createTableOnly = params.has("create-table-only");
         boolean ignoreDefaultValue = params.has("ignore-default-value");
+        boolean mergeSameSchema = params.getBoolean("merge-same-schema", true);
```

Comment:
use has Options will better？

Reply:
Currently, when the option `database test_db` is set in the following shell command, all tables in MySQL will be synchronized to a single database in Doris. The records from the `customer_1` and `customer_2` tables in MySQL will be synchronized into one Doris table `customer_1`. The option `merge-same-schema` is set to true by default to maintain compatibility with the behavior of older versions.